index.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 module.exports = function(fn) {
-  return function(event) {
-    event.stopPropagation();
-    return typeof fn === 'function' && fn(event);
-  };
-};
\ No newline at end of file
+  return typeof fn === 'function'
+    ? function(event) {
+        event.stopPropagation();
+        return fn(event);
+      }
+    : function() {
+        return false;
```

Comment:
@AndersCan Does this mean that passing a non-function will result in preventDefault? and stopPropagation. and everything else i guess. thoughts?

Reply:
@jonhni If I add an `event.stopPropagation();` before line 9 (`return false`) it should be equivalent to the old implementation. So yes, `return false` should equal a `preventDefault()`

Is `fn` supposed to be an optional callback? I think it makes more sense to not return its value (or any value at all).